src/passes/rejectUnsupportedFeatures.ts

```diff
@@ -59,12 +58,6 @@ export class RejectUnsupportedFeatures extends ASTMapper {
   visitErrorDefinition(node: ErrorDefinition, _ast: AST): void {
     throw new WillNotSupportError('User defined Errors are not supported', node);
   }
-  visitConditional(node: Conditional, _ast: AST): void {
```

Comment:
This should still reject non-void tuple conditionals as they aren't supported (right?). It should also reject tuple assignments that are children of anything other than an expression statement